Crash consistent VM restore points (preview)/README.md

```diff
@@ -0,0 +1,33 @@
+# What is a Crash consistent VM Restore Point? 
+A crash consistent VM restore point stores the VM configuration and point-in-time write-order consistent snapshots for all managed disks attached to a Virtual Machine. This is same as the status of the data in the VM after a power outage or a crash.
+
+## Get started
+You can now create multi-disk crash consistent restore points for your Azure VMs and use these restore points for backup and/or disaster. Crash consistent VM restore points are available in the following regions: EAST US EUAP and West Central US. If you want know more abour VM restore points. Please review our [VM restore points public documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-create-restore-points).
```

Comment:
Can we drop 'West Central US' for now, since we don't have support in that region?

Reply:
Done